pkg/detectors/gitlab/v1/gitlab.go

```diff
@@ -32,7 +32,7 @@ func (Scanner) DefaultEndpoint() string { return "https://gitlab.com" }
 
 var (
 	defaultClient = common.SaneHttpClient()
-	keyPat        = regexp.MustCompile(detectors.PrefixRegex([]string{"gitlab"}) + `\b([a-zA-Z0-9\-=_]{20,22})\b`)
+	keyPat        = regexp.MustCompile(detectors.PrefixRegex([]string{"gitlab"}) + `\b([a-zA-Z0-9\-_]{20,22})\b`)
```

Comment:
Similar changes are required for Gitlab v2 detector if we are sure that Gitlab token does not have `=` character in them.

Reply:
The reason I said is that we should be consistent as they key pattern is same. As mentioned in the description that `=` is not seen as part of the token. If we are updating regex for v1 I believe we should do it for v2 as well.